package.json

```diff
@@ -24,9 +27,18 @@
     "babel": "6.5.2",
     "babel-eslint": "6.1.2",
     "eslint": "3.4.0",
+    "husky": "^0.14.3",
     "istanbul": "0.4.5",
+    "lint-staged": "^6.0.0",
     "marked-toc": "0.3.0",
     "mocha": "3.0.2",
+    "prettier": "^1.9.2",
```

Comment:
Maybe we should put these three (husky, lint-staged, and prettier) to `devDependencies`?

Reply:
aren't they? 

![](https://cdn.zapier.com/storage/photos/312b17e590ab5788943850eb6d9b7e69.png)

Reply:
My bad 🤦‍♂️. The change looks good!